File: Java/Lista_05(Matrizes)/Cada_Linha.java
import java.util.Scanner;

public class Cada_Linha {

    public static void main(String[] args) {

        Scanner s = new Scanner(System.in);

        System.out.print("Qual a ordem da matriz? ");
        int n = s.nextInt();

        int mat[][] = new int[n][n];

        for (int i = 0; i < n; i++) {
            for (int j = 0; j < n; j++) {
                System.out.print("Elemento [" + i + "][" + j + "]: ");
                mat[i][j] = s.nextInt();
            }
        }

        int maior[] = new int[n];

        for (int i = 0; i < n; i++) {
            maior[i] = mat[i][0];
            for (int j = 0; j < n; j++) {
                if (mat[i][j] > maior[i]) {
                    maior[i] = mat[i][j];
                }
            }
        }

        System.out.println("MAIOR ELEMENTO DE CADA LINHA:");

        for (int i = 0; i < n; i++) {
            System.out.println(maior[i]);
        }

        s.close();

    }
}

File: Python/Lista_05(Matrizes)/Cada_Linha.py
n = int(input("Qual a ordem da matriz? "))

mat = [[0 for x in range(n)] for x in range(n)]

for i in range(0,n):
    for j in range(0,n):
        mat[i][j] = int(input(f"Elemento [{i},{j}]: "))

print("NAIOR ELEMENTO DE CADA LINHA:")

for i in range(0,n):
    maior = mat[i][0]
    for j in range(0,n):
        if mat[i][j]>maior:
            maior = mat[i][j]
    print(maior)